### PR TITLE
Release fixes

### PR DESCRIPTION
**Pull request by wasade**
I think this covers the necessary changes. Do not merge until we're green on travis as this PR introduces a default non-localhost `REDBIOM_HOST` necessitating logic to complain appropriately if testing off localhost.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,5 +26,7 @@ install:
   - alias md5=md5sum
   - pip install -e . --no-deps
 script:
-  - flake8 --ignore=E731 redbiom 
+  - flake8 --ignore=E731 redbiom
+  - redbiom summarize contexts  # will return a nonzero exit status if it cannot communicate with the default host
```
Comment (antgonza) [inline .travis.yml]: I don't get this, you are testing if it can communicate (which it will fail and stop execution) and then change REDBIOM_HOST? Would you mind explaining why is this required?

**Comment by wasade** [comment]
Verify that the default host that is read only
(qiita) works, and then switch to a host which is mutable so we can execute
the test suite

File: .travis.yml (hunk 1 above)
Comment (antgonza) [inline .travis.yml]: just to confirm, does this mean that it will fail if qiita is down?

**Comment by wasade** [comment]
Yes. But that shouldn't happen, right?

File: .travis.yml (hunk 1 above)
Comment (antgonza) [inline .travis.yml]: right, just wanted to be sure.

**Comment by wasade** [comment]
Thanks!
